index.html

```diff
@@ -47,9 +47,8 @@
         for title in response.css('h2.entry-title'):
             yield {'title': title.css('a ::text').extract_first()}
 
-        next_page = response.css('div.prev-post > a ::attr(href)').extract_first()
-        if next_page:
-            yield scrapy.Request(response.urljoin(next_page), callback=self.parse)
+        for next_page in response.css('div.prev-post > a'):
+            yield response.follow(next_page, self.parse)
```

Comment:
@redapple only you left to approve this PR. feel free to merge if you approve it.